### ISSUE TITLE
code folding state lost when cutting and pasting folded code

### ISSUE THREAD

**Issue by danyn** [adobe/brackets#13014]
### Description

code folding state does not maintain when pasting code that has been folded.

### Steps to Reproduce

1. fold the elements to be copied
2. copy then to the clipboard
3. paste them in desired location

**Expected behavior:** Expected the code to remain folded

**Actual behavior:**  The code expands upon being pasted

### Versions

Windows 10 Pro on intel core i5 M20
Release 1.8 build 1.8.0-17108 (alf_localization_release_1.8 3af64fae4) 
build timestamp: Wed Nov 02 2016 12:07:29 GMT+0000

**Comment by TimBibb** [comment]
Yup, that's still happening with me as well. It's happening on most languages that I have tested. I'm going to look for the issue and try to fix it.
